tests/e2e.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+#
+# E2E tests for kpt-functions-catalog.
+
+set -eo pipefail
+
+REPO="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. >/dev/null 2>&1 && pwd)"
+TAG=${TAG:-dev}
+NODOCKER=${NODOCKER:-}
+HELM_USAGE_SNIPPET="Render chart templates locally"
+
+############################
+# Test framework
+############################
+
+function testcase() {
+  echo "testcase: ${1}"
+  tmp=$(mktemp -d "/tmp/e2e.${1}.XXXXXXXX")
+  cp -r "${REPO}"/example-configs "${tmp}"
+  cp -r "${REPO}"/sh/test-data/* "${tmp}"
+  cd "${tmp}"
+}
+
+function fail() {
+  echo "FAIL: " "$@"
+  exit 1
+}
+
+function assert_contains_string() {
+  content="$(<"$1")"
+  grep -q "$2" "$1" || fail "String $2 not contained in: ${content}"
+}
+
+function assert_dir_exists() {
+  [[ -d $1 ]] || fail "Dir not exist: $1"
+}
+
+############################
+# Docker Tests
+############################
+
+[[ -z ${NODOCKER} ]] || {
```

Comment:
Remove this and the variable as this is only does docker testing

Reply:
Done